### DISCUSSION TITLE
What's everyone's favorite mod they have made (or are making if able to share if in WIP) for FNF (specifically using Modding+, but can be in general)?

### DISCUSSION THREAD

**Discussion by RebatemanYT** [FunkinModdingPlus/ModdingPlus#214] [General]
Just was wondering.
Btw mine rn is my modpack I am working on, but of that modpack, I feel like either the covers of Week 6 or the High Quality Rip of the Roses cover (joke of World 3A - Dora And Friends: Fantastic Flight) is the best part of the pack so far.

**Comment by Mackery6969** [comment]
Mine has to be vs bob retweaked

**Comment by Djw1213** [comment]
Mine is vs Eddsworld Matt
